test/unit/base-connection-pool.test.js

```diff
@@ -141,7 +141,7 @@ test('API', t => {
           http: {
             publish_address: '127.0.0.1:9200'
           },
-          roles: ['master', 'data', 'ingest']
+          roles: ['cluster_manager', 'data', 'ingest']
```

Comment:
Just a note - changing all the test roles in this file is actually removing test coverage for the `master` role. Ideally, we should still test the deprecated role until we actually remove support for it.

Reply:
Yeah, so the purpose of changing some tests is to make sure both `cluster_manager` and `master` are working. Instead of writing very similar unit tests for these two roles, I decide to modify some current unit tests, especially if there are multiple nodes. Would like more discussion here.

Reply:
I think confirming that both options work the same is worthwhile in any of the tests that currently use `master`.